cpp/ql/src/experimental/Security/CWE/CWE-359/PrivateCleartextBufferWrite.ql

```diff
@@ -0,0 +1,33 @@
+/**
+ * @name Cleartext storage of private data in buffer
+ * @description Storing private data in cleartext can expose it
+ *              to an attacker.
+ * @kind path-problem
+ * @problem.severity warning
+ * @precision medium
```

Comment:
Please delete the `@precision` line unless there is a good reason for assigning this particular precision. See step 4 of [supported-queries.md](https://github.com/github/codeql/blob/c0ebf12ab0bfcde6482cd2f08c913c6ac8b59a4d/docs/supported-queries.md) for the process if you'd like to do this later.

Reply:
Sounds great, just autoformatted the qll file you mentioned and removed these tags on both ql files.